sortdb/metadata.csv

```diff
@@ -0,0 +1,29 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration
+sortdb.stats.total_requests,GAUGE,,task,,Total number of requests,0,sortdb
+sortdb.stats.total_seeks,GAUGE,,task,,Total number of seeks,0,sortdb
+ sortdb.stats.get_requests ,GAUGE,,task,,Total number of get requests,0,sortdb
+ sortdb.stats.get_hits ,GAUGE,,task,,Total number of get hits,0,sortdb
+ sortdb.stats.get_misses ,GAUGE,,task,,Total number of get misses,-1,sortdb
+ sortdb.stats.get_requests.avg ,GAUGE,,task,,Average get requests received,0,sortdb
+ sortdb.stats.get_requests.95percentile ,GAUGE,,task,,95th percentile of get requests received,0,sortdb
+ sortdb.stats.get_requests.99percentile ,GAUGE,,task,,99th percentile of get requests,0,sortdb
+ sortdb.stats.mget_requests ,GAUGE,,task,,Total number of mget requests,0,sortdb
+ sortdb.stats.mget_hits ,GAUGE,,task,,Total number of mget hits,0,sortdb
+ sortdb.stats.mget_misses ,GAUGE,,task,,Total number of mget misses,-1,sortdb
+ sortdb.stats.mget_requests.avg ,GAUGE,,task,,Average mget requests,0,sortdb
+ sortdb.stats.mget_requests.95percentile ,GAUGE,,task,,95th percentile of mget requests,0,sortdb
+ sortdb.stats.mget_requests.99percentile ,GAUGE,,task,,99th percentile of mget requests,0,sortdb
+ sortdb.stats.fwmatch_requests ,GAUGE,,task,,Total number of fwmatch requests,0,sortdb
+ sortdb.stats.fwmatch_hits ,GAUGE,,task,,Total number of fwmatch hits,0,sortdb
+ sortdb.stats.fwmatch_misses ,GAUGE,,task,,Total number of fwmatch misses,-1,sortdb
+ sortdb.stats.fwmatch_requests.avg ,GAUGE,,task,,Average fwmatch requests,0,sortdb
+ sortdb.stats.fwmatch_requests.95percentile ,GAUGE,,task,,95th percentile of fwmatch requests,0,sortdb
+ sortdb.stats.fwmatch_requests.99percentile ,GAUGE,,task,,99th percentile of fwmatch requests,0,sortdb
+ sortdb.stats.range_requests ,GAUGE,,task,,Total number of range requests,0,sortdb
+ sortdb.stats.range_hits ,GAUGE,,task,,Total number of range hits,0,sortdb
+ sortdb.stats.range_misses ,GAUGE,,task,,Total number of range misses,-1,sortdb
+ sortdb.stats.range_requests.avg ,GAUGE,,task,,Average range requests,0,sortdb

```

Comment:
This is in microseconds, as is the 95th and 99th percentiles.
https://github.com/jehiah/sortdb/blob/master/src/cmd/sortdb/http.go#L272

Addtionally, isn't lower better? (-1 instead of 0 in orientation)